### DISCUSSION TITLE
Azure Hybrid Use Benefit calculations seem off

### DISCUSSION THREAD

**Discussion by lundejd** [microsoft/finops-toolkit#709] [General]
I finally got around to reviewing the Hybrid Benefit tab and am immediately confused by the numbers. For example, we have a Standard_D2ds_v5 server that has 2 vCPUs. On the bottom that same server is listed but the column vCPUs says 16. I have another of the same Standard_D2ds_v5 that has vCPUs of 2, which is correct, and AHB vCPUs of 8 which is correct.

This is only the beginning. Let's say I wanted to license a 16 vCPU server in Azure with AHUB, I would only need to use 16 core licenses to do it. The chart shows 64 and a usage quantity of 24. 

I would think AHUB should look like this:

vCPUs.  AHB vCPUs
1                   8
2                  8
8                   8
10                 16
16                 16
20                24

and so on. minimum 8, multiples of 8. vCPU systems that don't divide into 8 end up with some "wasted" AHUB cores.

AHB vCPUs that are enabled should equal UsageQuantity. If I have an 8 vCPU machine enabled for AHUB it should use 8 vCPUs. UsageQuantity seems to be 24 for most.

Here is a screen shot with the same size server, the first has accurate info but the following do not.

<img width="772" alt="Screenshot 2024-05-20 at 8 42 30 AM" src="https://github.com/microsoft/finops-toolkit/assets/8152933/fb95a775-c874-42b7-b9c5-2d05434f857c">

**Comment by flanakin** [comment]
Moved this to an issue so we can track the bug. Thanks for reporting!
